compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -83,6 +83,7 @@ declare_lint_pass! {
         PUB_USE_OF_PRIVATE_EXTERN_CRATE,
         REDUNDANT_IMPORTS,
         REDUNDANT_LIFETIMES,
+        UNNECESSARY_TRANSMUTATION,
```

Comment:
please call it `unnecessary_transmute`, not `transmutation`.

Reply:
`transmute`

Reply:
Plural, [right](https://rustc-dev-guide.rust-lang.org/diagnostics.html?highlight=plural#lint-naming)?  So `unnecessary_transmutes`?